style/scala/README.md

```diff
@@ -0,0 +1,10 @@
+# Scala
+
+* Avoid `var`
+* Avoid implicit conversions
+* Avoid wildcard imports except for implicits
+* Avoid `null` except when necessary for Java compatibility
+* Prefer `IO` to `Future`
```

Comment:
I'm guessing that `IO` here refers to either Cats or Monix, but less so Scalaz. Worth annotating with pointers, or are you purposefully keeping this more general?

Reply:
I clarified this a bit in c4ccfa1. I basically want to avoid `Future` and `cats.effect.IO` is an easy choice for many apps, but `monix.eval.Task` or `zio.IO` are also good.